packages/keycodes/src/index.js

```diff
@@ -65,7 +65,7 @@ const modifiers = {
  */
 export const rawShortcut = mapValues( modifiers, ( modifier ) => {
 	return ( character, _isApple = isAppleOS ) => {
-		return [ ...modifier( _isApple ), character.toLowerCase() ].join( '+' );
+		return [ ...modifier( _isApple ), character.toLowerCase() ].filter( ( key ) => key !== '' ).join( '+' );
```

Comment:
What was the reason that filtering is introduced?

Reply:
This is because when we are passing second argument to `pressWithModifier` there is no need to pass it to `rawShortcut` also, because `key` is used only in  here 'await page.keyboard.press( key );`

Reply:
I think it might be tidier to export the modifiers const, and write a function that maps the keys exactly as they're needed within the e2e/utils (instead of using rawShortcut in tests).

That would also allow the possibility of adding extra combinations to the list of modifiers that are only used in tests (e.g. the shift+control word selection modifiers) and the production implementation wouldn't need to be touched.